src/Features/Core/Portable/MoveToNamespace/AbstractMoveToNamespaceService.cs

```diff
@@ -190,7 +190,7 @@ private static async Task<ImmutableArray<ISymbol>> GetMemberSymbolsAsync(Documen
             {
                 case TNamespaceDeclarationSyntax namespaceNode:
                     var containerSymbol = (INamespaceSymbol)semanticModel.GetDeclaredSymbol(container, cancellationToken);
-                    return containerSymbol.GetMembers().SelectAsArray(m => (ISymbol)m);
+                    return containerSymbol.GetMembers().Where(el => el.IsFromSource()).SelectAsArray(m => (ISymbol)m);
```

Comment:
based on the code below, it seems like we want the syntactic members that are inside 'container'.  CAn you use syntaxFacts.GetMembersOfBaseNamespaceDeclaration like you do GetMembersOfCompilationUnit below?  Thanks!